pdns/dnsdistdist/docs/rules-actions.rst

```diff
@@ -84,26 +86,47 @@ Rule Generators
 
 .. function:: addDelay(DNSrule, delay)
 
+  .. deprecated:: 1.2.0
+
   Delay the query for ``delay`` milliseconds before sending to a backend.
+  This function is deprecated as of 1.2.0, please use instead:
+
+    addAction(DNSRule, DelayAction(delay))
 
   :param DNSRule: The DNSRule to match traffic
   :param int delay: The delay time in milliseconds.
 
 .. function:: addDisableValidationRule(DNSrule)
 
+  .. deprecated:: 1.2.0
+
   Set the CD (Checking Disabled) flag to 1 for all queries matching the DNSRule.
-  Using this function is equal to using the :func:`DisableValidationAction`.
+  Using this function is equal to using the :func:`DisableValidationAction` action.
 
 .. function:: addDomainBlock(domain)
 
+  .. deprecated:: 1.2.0
+
   Drop all queries for ``domain`` and all names below it.
+  Deprecated as of 1.2.0, please use instead:
+
+    addAction(domain, DropAction())
 
   :param string domain: The domain name to block
 
 .. function:: addDomainSpoof(domain, IPv4[, IPv6])
               addDomainSpoof(domain, {IP[,...]})
 
+  .. deprecated:: 1.2.0
```

Comment:
All these deprecated things could have the line "Will be removed in 1.3.0" in there as well?